### PR TITLE
Feat/25.0.0

### LINKED ISSUE #667: Handle metrics in heatmap representation
- [ ] Represent heatmaps using `circle-blur`
- [ ] Deprecate the current heatmap that represents counts only
- [ ] Allow interpolation of `opacity`

### LINKED ISSUE #916: Store the preview id in the config.json
-  [ ] use the updated schema for config.json gisaia/ARLAS-wui-toolkit#718
-  [ ] if no preview exists, create a new one and store the id in config.json
-  [ ] if a preview exists already, update the resource and keep the same uuid.

### LINKED ISSUE #927: ARLAS Builder: building a heatmap deselect the geometry
### Description of the bug

I try to create a Circle heatmap with the builder on the Course collection
- I choose a `cluster` visualization
- i select `courses`as the collection
- i select `arrival.location` as the aggregation geofield
- i select `Tile grid` for the grid type
- i select `Medium` for the granularity
- i select `Aggregated geometry`
- i select `Data centroid`
image

Note: arlas stacks 4 tests with IAM

### What was the error message shown?

```JavaScript
See "Steps to reproduce"
```

### Steps to reproduce

Then I go to the `stye` panel
- I choose `Circle heatmap`
Then I move back to the  `Geometry` panel
- i see that the aggregation geofield is deselected with an error message
- i see that `Data centroid` became `Cell's center`

image

### If the issue was noticed in the demo workspace, please specify the collection and the field. Otherwise, please link the 2 ARLAS config files.

_No response_

### Which ARLAS component(s) are concerned by the bug?

Other

### If you chose 'Other', please specify the part(s) of ARLAS concerned by the bug.

Builder

### ARLAS version

25.0.0-beta.2

### What browsers are you seeing the problem on?

Chrome

### If you chose 'Other', please specify the browser(s) concerned by the bug.

_No response_

### LINKED ISSUE #931: Saving the preview in Postgres database triggers a 500 error
When using persistence whith Postgres database, it demands to store the resource as a stringified JSON
However, the preview is stored as a base64 image which is the source of the error mentionned above

### LINKED ISSUE #933: Applying timeline filter [use_timeline_filter] with IAM throws an error because org is missing

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
- Fix #933 
- Fix #931 
- Fix #927 
- Fix #916 
- Fix #667

### CODE REVIEW

File: src/app/services/main-form-manager/config-map-export-helper.ts [hunk 1]
```diff
@@ -35,13 +35,16 @@ import {
   Layer,
   Layout,
   MapConfig,
-  Paint,
+  Paint, PaintValue,
   SELECT_LAYER_PREFIX
 } from './models-map-config';
+import {InterpolatedProperty, ModesValues} from '@shared/interfaces/config-map.interfaces';
+import {CIRCLE_HEATMAP_RADIUS_GRANULARITY} from '@shared-models/circle-heat-map-radius-granularity';
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-map-export-helper.ts:42]: Missing spaces

File: src/app/services/main-form-manager/config-map-export-helper.ts [hunk 2]
```diff
@@ -231,6 +245,19 @@ export class ConfigMapExportHelper {
     return layer;
   }
 
+  /**
+   * set the correct layer type before we save it.
+   * @param geometryType
+   */
+  public static getLayerType(geometryType: GEOMETRY_TYPE): GEOMETRY_TYPE | string {
+    /** we change the type of circle heat map  to keep the compatibility with mapbox **/
+    if(geometryType === GEOMETRY_TYPE.circleHeat){
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-map-export-helper.ts:254]: Missing space

File: src/app/services/main-form-manager/config-map-export-helper.ts [hunk 3]
```diff
@@ -166,6 +169,17 @@ export class ConfigMapExportHelper {
       collection,
       collectionDisplayName
     };
+
+    // with this prop we ll be able to restore the good geomType when we ll reload the layer;
+    if (metadata.hasOwnProperty('hiddenProps')) {
+      delete metadata['hiddenProps'];
+    }
+
+    if(modeValues.styleStep.geometryType === GEOMETRY_TYPE.circleHeat) {
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-map-export-helper.ts:178]: missing space

**Review by sebbousquet — APPROVED** [review]

File: src/app/services/main-form-manager/config-map-export-helper.ts (hunk 3 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-map-export-helper.ts]: fixed

File: src/app/services/main-form-manager/config-map-export-helper.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-map-export-helper.ts]: fixed

File: src/app/services/main-form-manager/config-map-export-helper.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-map-export-helper.ts]: fixed

**Pull request merged**